monitor/README.md

```diff
@@ -121,7 +121,22 @@ Example of an API
 ```
```

Comment:
Just discovered that the information about `GET /stuckTransfers` is not applicable anymore since related to call . Please remove it and the file `checkWorker3.js`. By doing this we could also close https://github.com/poanetwork/tokenbridge/issues/79

Reply:
Done.

Reply:
thanks

Reply:
please remove `checkWorker3.js` as well from the repo.

Reply:
It has already been removed.

Reply:
Oh. Great! It seems that `stuckTransfers.js` is not used anymore so it could be also removed.

Reply:
Done - removed `stuckTransfers.js`